From 0d2b624f1e24240a1c4e651aa1cfe9a8dd10a573 Mon Sep 17 00:00:00 2001
From: "Carsten Haitzler (Rasterman)" <raster@rasterman.com>
Date: Sat, 15 Dec 2018 16:19:01 +0000
Subject: [PATCH] evas gl - make GLintptr etc. also ndefed for GL_VERSION_1_5
 fix typedef

It seems that GL_VERSION_1_5 define == these provided by gl already. At
least reading the mesa headers I do, so this should fix T7502

@fix
---
 src/lib/evas/Evas_GL.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/lib/evas/Evas_GL.h b/src/lib/evas/Evas_GL.h
index b6b642400f..4f67b1695f 100644
--- src/lib/evas/Evas_GL.h
+++ src/lib/evas/Evas_GL.h
@@ -4272,9 +4272,11 @@ typedef signed int       GLfixed;      // Changed khronos_int32_t
 
 #ifndef GL_ES_VERSION_2_0
 /* GL types for handling large vertex buffer objects */
-#include <stddef.h>
+# ifndef GL_VERSION_1_5
+#  include <stddef.h>
 typedef ptrdiff_t GLintptr;     // Changed khronos_intptr_t
 typedef ptrdiff_t GLsizeiptr;   // Changed khronos_ssize_t
+# endif
 #endif
 
 /* Some definitions from GLES 3.0.
